src/main/flight/gps_rescue.c

```diff
@@ -693,19 +689,19 @@ void initialiseRescueValues (void)
 void gpsRescueUpdate(void)
 // runs at gpsRescueTaskIntervalSeconds, and runs whether or not rescue is active
 {
+    static uint16_t gpsStamp = 0;
+    bool newGpsData = gpsHasNewData(&gpsStamp);
+
     if (!FLIGHT_MODE(GPS_RESCUE_MODE)) {
         rescueStop(); // sets phase to RESCUE_IDLE; does nothing else.  RESCUE_IDLE tasks still run.
     } else if (FLIGHT_MODE(GPS_RESCUE_MODE) && rescueState.phase == RESCUE_IDLE) {
         rescueStart(); // sets phase to rescue_initialise if we enter GPS Rescue mode while idle
-        rescueAttainPosition(); // Initialise basic parameters when a Rescue starts (can't initialise sensor data reliably)
+        rescueAttainPosition(newGpsData); // Initialise basic parameters when a Rescue starts (can't initialise sensor data reliably)
```

Comment:
(this overloading of rescueAttainPosition is strange. It will be called second time in this function. And it does not handle GPS during RESCUE_INITIALIZE)
```suggestion
        rescueAttainPosition(false); // Initialise basic parameters when a Rescue starts (can't initialise sensor data reliably)
```

Reply:
It's just part of the dumb original way GPS Rescue was written.  This isn't something I changed.  I do try to improve it in the next PR.
When the rescue first starts, phase is idle, so the first call to `rescueAttainPosition` only gets to do the `RESCUE_IDLE` phase actions, which return zeroes for the rescue angles, then returns. This is likely unnecessary.  The phase then gets set to something else and then 
Thereafter `rescueAttainPosition` is only called at the end once the settings are determined according to the phase.
This needs re-working and I do so in the upcoming GPS PR.  Let's worry about it there. There's a lot do to do there.
Have set to false here.